### ISSUE TITLE
Auto-stil whitelist

### ISSUE THREAD

**Issue by ghost** [Z3r0byte/Magistify#44]
De auto-stil functie wordt zo nu en dan constant ingeschakeld, omdat mijn school vakanties en feestdagen ook als afspraken in Magister zet, waardoor Magistify mijn telefoon de hele dag (of week) op stil zet, en ik het niet uit kan zetten.
Misschien zou je een filter kunnen toevoegen, waardoor de auto-stil functie niet aangaat als de afspraak de hele dag duurt. Dit zou je ook optioneel kunnen maken.

**Comment by StanNL** [comment]
Volgens mij is dit bericht niet helemaal aangekomen ;)
Heb je e-mailnotificaties aan staan?

**Comment by Z3r0byte** [comment]
Nee, en op de een of andere manier stond deze ook niet in het Github dashboard, maar bedankt voor het melden hiervan, ik zal kijken wat ik eraan kan doen. ;)

**Comment by StanNL** [comment]
@Z3r0byte FastHub kan je meldingen sturen

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
@StanNL  weet ik, maar dit werkt op de een of andere manier niet altijd even goed

**Comment by StanNL** [comment]
@Z3r0byte Ooh raar, misschien moet ik je ervoor mentionen

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
@StanNL , ik heb net even in de instellingen de notificaties aan en uit gezet, misschien dat dat helpt.

**Comment by Z3r0byte** [comment]
Ik zie trouwens dat je een logo hebt gemaakt, nice!

**Comment by StanNL** [comment]
@Z3r0byte Ja dat heeft voor mij wel geholpen in ieder geval

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by StanNL** [comment]
@Z3r0byte Thanks, even club een Material Icon Generator gebruikt ;)

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
De notificaties doen het nu! :)

**Comment by Z3r0byte** [comment]
Ik gebruik voor de meeste mijn app-logo's ook een Material Icon generator, hoor ;)

**Comment by StanNL** [comment]
@Z3r0byte Dat is mooi, maar nu werken die van mij niet meer ;)

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
@StanNL , misschien komt dat omdat ik je niet had gementioned? ;)

**Comment by StanNL** [comment]
@Z3r0byte Ik weet het niet, ik heb hem nu alweer niet ontvangen. Misschien komt het omdat ik de app nog in de voorgrond had, want in het notificaties tabblad in de app (en GitHub) zelf stond de melding er wel bij

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
@StanNL , ik weet niet precies waar dat aan ligt, ik gebruik deze app nog niet zo lang. Gelukkig werken ze bij mij nu wel goed.

**Comment by StanNL** [comment]
@Z3r0byte Ach, zo belangrijk vind ik de meldingen nu ook weer niet. Heb je al gezien dat 3.0.0 vandaag is uitgebracht?

_Sent from my Samsung kminilte using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Z3r0byte** [comment]
Ja, ik heb hem net gedownload

**Comment by Z3r0byte** [comment]
Ik heb hem gereopend omdat een "white-list" wel op het programma staat ;)

**Comment by ghost** [comment]
Ooh mooi!
